.. role:: raw-html(raw)
   :format: html

References
==========

[velocyto] La Manno, G., Soldatov, R., Zeisel, A., Braun, E., Hochgerner, H., Petukhov, V., Lidschreiber, K., Kastriti, M.E., Lönnerberg, P., Furlan, A., et al. (2018). RNA velocity of single cells. Nature 560, 494–498. :raw-html:`<a href="https://doi.org/10.1038/s41586-018-0414-6" target="_blank">https://doi.org/10.1038/s41586-018-0414-6</a>`.

[scVelo] Bergen, V., Lange, M., Peidli, S., Wolf, F.A., and Theis, F.J. (2020). Generalizing RNA velocity to transient cell states through dynamical modeling. Nat Biotechnol 38, 1408–1414. :raw-html:`<a href="https://doi.org/10.1038/s41587-020-0591-3" target="_blank">https://doi.org/10.1038/s41587-020-0591-3</a>`.

[VeloAE] Qiao, C., and Huang, Y. (2021). Representation learning of RNA velocity reveals robust cell transitions. Proc. Natl. Acad. Sci. U.S.A. 118, e2105859118. :raw-html:`<a href="https://doi.org/10.1073/pnas.2105859118" target="_blank">https://doi.org/10.1073/pnas.2105859118</a>`.

[Dynamo] Qiu, X., Zhang, Y., Martin-Rufino, J.D., Weng, C., Hosseinzadeh, S., Yang, D., Pogson, A.N., Hein, M.Y., Hoi (Joseph) Min, K., Wang, L., et al. (2022). Mapping transcriptomic vector fields of single cells. Cell 185, 690-711.e45. :raw-html:`<a href="https://doi.org/10.1016/j.cell.2021.12.045" target="_blank">https://doi.org/10.1016/j.cell.2021.12.045</a>`.

[VeloVAE] Gu, Y., Blaauw, D.T., and Welch, J. (2022). Variational Mixtures of ODEs for Inferring Cellular Gene Expression Dynamics. In Proceedings of the 39th International Conference on Machine Learning Proceedings of Machine Learning Research., K. Chaudhuri, S. Jegelka, L. Song, C. Szepesvari, G. Niu, and S. Sabato, eds. (PMLR), pp. 7887–7901.

[MultiVelo] Li, C., Virgilio, M.C., Collins, K.L., and Welch, J.D. (2023). Multi-omic single-cell velocity models epigenome–transcriptome interactions and improves cell fate prediction. Nat Biotechnol 41, 387–398. :raw-html:`<a href="https://doi.org/10.1038/s41587-022-01476-y" target="_blank">https://doi.org/10.1038/s41587-022-01476-y</a>`.

[UniTVelo] Gao, M., Qiao, C., and Huang, Y. (2022). UniTVelo: temporally unified RNA velocity reinforces single-cell trajectory inference. Nat Commun 13, 6586. :raw-html:`<a href="https://doi.org/10.1038/s41467-022-34188-7" target="_blank">https://doi.org/10.1038/s41467-022-34188-7</a>`.

[DeepVelo (VAE-based)] Chen, Z., King, W.C., Hwang, A., Gerstein, M., and Zhang, J. (2022). DeepVelo: Single-cell transcriptomic deep velocity field learning with neural ordinary differential equations. Science Advances 8, eabq3745. :raw-html:`<a href="https://doi.org/10.1126/sciadv.abq3745" target="_blank">https://doi.org/10.1126/sciadv.abq3745</a>`.

[cellDancer] Li, S., Zhang, P., Chen, W., Ye, L., Brannan, K.W., Le, N.-T., Abe, J., Cooke, J.P., and Wang, G. (2024). A relay velocity model infers cell-dependent RNA velocity. Nature Biotechnology 42, 99–108. :raw-html:`<a href="https://doi.org/10.1038/s41587-023-01728-5" target="_blank">https://doi.org/10.1038/s41587-023-01728-5</a>`.

[PhyloVelo] Wang, K., Hou, L., Wang, X., Zhai, X., Lu, Z., Zi, Z., Zhai, W., He, X., Curtis, C., Zhou, D., et al. (2024). PhyloVelo enhances transcriptomic velocity field mapping using monotonically expressed genes. Nature Biotechnology 42, 778–789. :raw-html:`<a href="https://doi.org/10.1038/s41587-023-01887-5" target="_blank">https://doi.org/10.1038/s41587-023-01887-5</a>`.

[veloVI] Gayoso, A., Weiler, P., Lotfollahi, M., Klein, D., Hong, J., Streets, A., Theis, F.J., and Yosef, N. (2024). Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells. Nature Methods 21, 50–59. :raw-html:`<a href="https://doi.org/10.1038/s41592-023-01994-w" target="_blank">https://doi.org/10.1038/s41592-023-01994-w</a>`.

[LatentVelo] Farrell, S., Mani, M., and Goyal, S. (2023). Inferring single-cell transcriptomic dynamics with structured latent gene expression dynamics. Cell Reports Methods 3, 100581. :raw-html:`<a href="https://doi.org/10.1016/j.crmeth.2023.100581" target="_blank">https://doi.org/10.1016/j.crmeth.2023.100581</a>`.

[DeepVelo (GCN-based)] Cui, H., Maan, H., Vladoiu, M.C., Zhang, J., Taylor, M.D., and Wang, B. (2024). DeepVelo: deep learning extends RNA velocity to multi-lineage systems with cell-specific kinetics. Genome Biol 25, 27. :raw-html:`<a href="https://doi.org/10.1186/s13059-023-03148-9" target="_blank">https://doi.org/10.1186/s13059-023-03148-9</a>`.

[TFvelo] Li, J., Pan, X., Yuan, Y., and Shen, H.-B. (2024). TFvelo: gene regulation inspired RNA velocity estimation. Nat Commun 15, 1387. :raw-html:`<a href="https://doi.org/10.1038/s41467-024-45661-w" target="_blank">https://doi.org/10.1038/s41467-024-45661-w</a>`.

[STT] Zhou, P., Bocci, F., Li, T., and Nie, Q. (2024). Spatial transition tensor of single cells. Nature Methods 21, 1053–1062. :raw-html:`<a href="https://doi.org/10.1038/s41592-024-02266-x" target="_blank">https://doi.org/10.1038/s41592-024-02266-x</a>`.
